frontend/OBSApp.cpp

```diff
@@ -1122,6 +1129,19 @@ bool OBSApp::OBSInit()
 
 	mainWindow = new OBSBasic();
 
+	#ifdef __APPLE__
+	QTimer::singleShot(0, this, [this]() {
+		id nsApp = ((id (*)(id, SEL))objc_msgSend)((id)objc_getClass("NSApplication"), sel_registerName("sharedApplication"));
+		BOOL isHidden = ((BOOL (*)(id, SEL))objc_msgSend)(nsApp, sel_registerName("isHidden"));
```

Comment:
Please avoid writing objc_msgSend, instead put this into a small helper function in platform-osx.mm.